iac-extensions/docker/src/test/resources/checks/PackageInstallationCacheCheck/mixed.dockerfile

```diff
@@ -42,6 +42,17 @@ RUN apt-get install nginx && aptitude clean
 RUN aptitude install nginx && apt clean
 RUN aptitude install nginx && apt-get clean
 
+# Compliant - direct package installation
+RUN apk add nginx.deb
+RUN apt-get install wget.deb
+RUN apk add wget.deb
+RUN apk add ./nginx.other
+RUN apt-get install ./wget.other
```

Comment:
Will `apt-get install wget.other` result in an issue? Generally speaking, it's not a valid name for a Debian package ([link](https://askubuntu.com/questions/1262528/what-are-usable-characters-are-for-package-names)) but maybe it can be tested for completeness